src/tasks/assign-user/assign-user-v2/task.json

```diff
@@ -71,6 +71,20 @@
       "type": "string",
       "required": true,
       "helpMarkDown": "Security role name or id to assign to the user"
+    },
+    {
+      "name": "ApplicationUser",
+      "label": "Application User",
+      "type": "boolean",
+      "required": false,
+      "helpMarkDown": "Specifies whether the input user is an application user. If business unit is not specified the application user will be added to the authenticated users business unit."
```

Comment:
2 spelling nits: 
- `If business unit is not specified>,< the...`
- `... authenticated user>'<s...`